### ISSUE TITLE
[Language] Controls missing role and state

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#65]
## Issue summary

The language select controls are missing their role. A role helps to describe what the control is, how to interact with it, and the expected outcome. Each are also missing their current state which is visually conveyed only using a checkmark icon. Screen readers do not convey the role or state which may cause a confusing user experience when interacting with the controls.

<details>
    <summary>Screenshots</summary>

<img width="372" alt="Screen Shot 2020-05-28 at 2 27 19 PM" src="https://user-images.githubusercontent.com/1392632/83179033-65934680-a0ef-11ea-96bf-a1cfffd7a3d2.png">

</details>

#### TSX

```tsx
// Language.tsx

<TouchableOpacity
  onPress={onPress}
>
  <Box
    …
  >
    <Text variant="bodyText" marginVertical="s" color="overlayBodyText">
      {text}
    </Text>
    {isActive && (
      <Box alignSelf="center">
        <Icon size={32} name="icon-check" />
      </Box>
    )}
  </Box>
</TouchableOpacity>
```

## Steps to reproduce

1. Load the app in the iOS simulator.
2. Open the Sheet view.
3. Activate the "Change language" control
4. Enable [VoiceOver screen reader](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) on macOS
5. Use the Virtual Cursor to navigate through the language controls

### Expected

- Controls to convey their role and state, both visually and audibly

### Actual

- Control role and state are not conveyed.

## Recommendation

1. Add the `accessibilityRole` prop, setting its value to `radio` to convey a radio button control.
2. Add the `accessibilityState` prop, setting its value to and object contain the `selected` boolean state. (<small>Note: the code below does <strong>not</strong> function as expected – please adjust as required.</small>)
3. Add the `accessibilityRole` prop to the `Box` container, setting its value to `radiogroup`.

#### TSX

```tsx
// Language.tsx

<TouchableOpacity
  onPress={onPress}
  accessibilityRole="radio"
  accessibilityState={{selected: {isActive}}}
>
  <Box
    …
  >
    <Text variant="bodyText" marginVertical="s" color="overlayBodyText">
      {text}
    </Text>
    {isActive && (
      <Box alignSelf="center">
        <Icon size={32} name="icon-check" />
      </Box>
    )}
  </Box>
</TouchableOpacity>

// …

<Box
  …
  accessibilityRole="radiogroup"
>
  <LanguageSelectItem
    …
  />
  <LanguageSelectItem
    …
  />
</Box>
```

## Specifications

- **Component:** CovidShield [Language]
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.1 Info and Relationships
- **Severity:** High
- **Effort:** Medium

**Comment by AbubakarSambo** [comment]
@svinkle , I have a PR open for this here: https://github.com/CovidShield/mobile/pull/82

**Comment by jordanAtShopify** [comment]
Fixed
